src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -655,42 +660,35 @@ def _handle_group_actions(
-            if not features.has(
-                "organizations:slack-sdk-webhook-handling", group.project.organization
-            ):
-                slack_client = SlackClient(integration_id=slack_request.integration.id)
-                try:
-                    private_metadata = orjson.loads(
-                        slack_request.data["view"]["private_metadata"],
-                    )
-                    slack_client.post(private_metadata["orig_response_url"], data=body, json=True)
-                except ApiError as e:
-                    logger.error("slack.action.response-error", extra={"error": str(e)})
 
-            else:
-                json_blocks = orjson.dumps(blocks.get("blocks")).decode()
-                view = View(**slack_request.data["view"])
-                try:
-                    private_metadata = orjson.loads(view.private_metadata)
-                    webhook_client = WebhookClient(private_metadata["orig_response_url"])
-                    webhook_client.send(
-                        blocks=json_blocks, delete_original=False, replace_original=True
-                    )
-                    logger.info(
-                        "slack.webhook.view_submission.success",
-                        extra={
-                            "integration_id": slack_request.integration.id,
-                            "blocks": json_blocks,
-                        },
-                    )
-                except SlackApiError as e:
-                    logger.error(
-                        "slack.webhook.view_submission.response-error", extra={"error": str(e)}
-                    )
+            # use the original response_url to update the link attachment
+            json_blocks = orjson.dumps(blocks.get("blocks")).decode()
+            view = View(**slack_request.data["view"])
+            try:
+                private_metadata = orjson.loads(view.private_metadata)
+                webhook_client = WebhookClient(private_metadata["orig_response_url"])
+                webhook_client.send(
+                    blocks=json_blocks, delete_original=False, replace_original=True
+                )
+                metrics.incr(
+                    SLACK_WEBHOOK_GROUP_ACTIONS_SUCCESS_DATADOG_METRIC,
+                    sample_rate=1.0,
+                    tags={"type": "submit_modal"},
+                )
+            except SlackApiError as e:
+                metrics.incr(
+                    SLACK_WEBHOOK_GROUP_ACTIONS_FAILURE_DATADOG_METRIC,
+                    sample_rate=1.0,
+                    tags={"type": "submit_modal"},
+                )
+                logger.error(
+                    "slack.webhook.view_submission.response-error",
+                    extra={
+                        "error": str(e),
+                        "integration_id": slack_request.integration.id,
+                        "organization_id": group.project.organization_id,
+                    },
+                )
```

Comment:
Thought for later: we repeat this logging and metrics in a lot of `except SlackApiError` blocks, right? We could try to abstract it into a utility function -- ideally a context manager that could take the place of the `try` block.